docs/meshcloud.project.md

```diff
@@ -40,38 +40,38 @@ Depending on the configuration of your meshStack implementation, you may be able
 
-If [4-eye-principle](meshstack.authorization.md#user-project-role-approval) is activated in your meshStack installation, an additional approval might be necessary, before a principal is actually assigned/remove/updated on a project.
+If [4-eye-principle](meshstack.authorization.md#user-project-role-approval) is activated in your meshStack installation, an additional approval might be necessary, before a user or group is actually assigned/remove/updated on a project.
 
 ### Assign user to a meshProject
 
-In the **Project Access Control** section you can find a type-ahead `Search for a customer user or group` field at the bottom. You search for users via their first and last name, email address and username. Groups can be found via their group name and identifier. You have to select the principal you want to assign in the dropdown. Also select a project role for the principal. Press the `+` to actually add this principal to the project. Note that only users with a valid account on the meshPanel can access your projects. A principal can be assigned multiple project roles on the same project.
+In the **Project Access Control** section you can find a type-ahead `Search for a customer user or group` field at the bottom. You search for users via their first and last name, email address and username. Groups can be found via their group name and identifier. You have to select the user or group you want to assign in the dropdown. Also select a project role. Press the `+` to add the user or group to the project. Note that only users with a valid account on the meshPanel can access your projects. You can assign multiple project roles to a user or a group on the same project.
 
 ### Project Roles
 
-Project roles grant principals a configured set of permissions in cloud platforms used by the project. Project roles can be configured according to exisiting requirements or policies, by default the following roles are used:
+Project roles grant a configured set of permissions in cloud platforms used by the project. Project roles can be configured according to exisiting requirements or policies, by default the following roles are used:
 
 - **Project Reader**: A read-only user, like a controller or similar.
 - **Project User**: A default user, like a developer, who can manage resources in the cloud platform.
 - **Project Admin**: An admin user, who can also change configurations of the project in the cloud platform.
 
 ### Update Project Role
 
-You can change the Project Role of a principal, by choosing a different role for the principal in the dropdown at the **Project Access** section, and clicking the **disc** icon afterwards to save the change.
+You can change the Project Role, by choosing a different role in the dropdown at the **Project Access** section, and clicking the **disc** icon afterwards to save the change.
 
-### Expiration of a principal assignment
+### Expiration of a user or group assignment
```

Comment:
d: When changing headlines, we always have to check for deep links. deep link work by using the headline in kebap-lower-case. So search for "expiration-of-a". If you find occurrences, you have to change the link accordingly. Sadly we don't get any compile errors for deep links. Errors only pop up when there is an issue in finding another md file in general.